src/overlays/actors/ovl_En_Ge2/z_en_ge2.c

```diff
@@ -228,14 +228,14 @@ void EnGe2_GetNextPath(EnGe2* this, PlayState* play) {
     Path* curPath;
     Path* nextPath;
     Vec3s* points;
-    u8 unk1;
+    u8 nextPathIndex;
 
     this->curPointIndex = 0;
 
     if (GERUDO_PURPLE_GET_PATH(&this->picto.actor) != GERUDO_PURPLE_PATH_NONE) {
         curPath = &play->setupPathList[GERUDO_PURPLE_GET_PATH(&this->picto.actor)];
-        unk1 = curPath->unk1;
-        nextPath = &play->setupPathList[unk1];
+        nextPathIndex = curPath->additionalPathIndex;
```

Comment:
What are your thoughts for `nextPathIndex` vs `additionalPathIndex`? Why go with `additionalPathIndex` over `nextPathIndex` for the main struct?

Reply:
https://github.com/zeldaret/mm/pull/823#discussion_r890676778
TLDR: Seems similar to the `additionalCsId` so like to mirror it.
(I also got the idea for customValue in that PR as well so thank you :) ).

Reply:
I haven't looked at the z_eventmgr in detail to know why it's called additionalCs there, but here `additionalPathIndex` references the next element in a linked list of `Path`s, so I don't understand the argument for not using `next` here.

Reply:
Tbh I don't really have a argument against `next`, just that it looks to be the same kind of implementation (pretty sure it is linked lists) so would like to mirror the naming to reflect this.

Reply:
Well, considering that three different people have suggested calling it nextPathIndex and you say that `additionalCsId` is implemented similarly, I have to ask why `additionalCsId` isn't `nextCsId`?

Reply:
`nextCsId` makes it sound like it's consecutive cutscenes played back-to-back which isn't always the case. It's a linked list, but `additionalCsId` makes it sound more optional which feels better to describe how the linked list in cutscenes are used

Reply:
For what it is worth. This seems to be similar to `paths` sometimes the `next` path isn't the actual next path to be used. It just happens to be the next `path` in the linked list.